distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Time_Zone.enso

```diff
@@ -50,7 +50,8 @@ parse_builtin text = @Builtin_Method "Time_Zone.parse_builtin"
    "Europe/Paris".
 @Builtin_Type
 type Time_Zone
-    ## The system default timezone.
+    ## ICON date_and_time
+       The system default timezone.
 
        > Example
          Get the system default timezone.
```

Comment:
Completely unrelated to this PR, but looking at the examples, I'm wondering if such trivial ones should be kept - given the component browser allows to easily create a node like the one shown here and the example description is just repeating the method description - it seems rather redundant. I guess such simple methods maybe shouldn't have examples at all? It feels like they only make sense if we can show some interesting usage.